Sources/OpenTelemetry/OTelSignal.swift

```diff
@@ -11,8 +11,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-import OpenTelemetry
-
-extension OTel.SpanID {
-    static let stub = OTel.SpanID(bytes: (1, 2, 3, 4, 5, 6, 7, 8))
+/// The supported instrumentation signals.
+public enum OTelSignal: String {
```

Comment:
I think Franz commented on this -- may be safer to do as a few static properties -- enums are horrible to evolve since we don't have "not frozen enums"

Reply:
I removed the type for now since it's not (yet) used anyways. I'll keep the static lets in mind for the future though 🙏